test_e2e/test_user_journey.py

```diff
@@ -0,0 +1,15 @@
+from time import sleep
+
+from selenium.webdriver.common.by import By
+
+
+def test_page_loads(driver, app_with_temp_board):
+    driver.get("http://127.0.0.1:5000/")
+    new_item = "Add another e2e test"
+
+    elem = driver.find_element(By.ID, "new_item")
+    elem.send_keys(new_item)
+    elem.submit()
+    sleep(1)
```

Comment:
Minor, but sleeps are often a (potentially necessary) smell; a few isn't a real issue but if they end up slowing down your test runs considerably (and as the volume of tests mount, it's quite feasible they do!) then they do harm the quick feedback loop we ideally want.
Here, I found removing the sleep did break the test 🙁 but I was able to swap this for `driver.implicitly_wait(1)` which has the advantage of being a "wait _up to_ 1 second" - you can set it once at the start of the test & it'll be assumed for all future finds